subprojects/wrapper/src/main/java/org/gradle/wrapper/Install.java

```diff
@@ -259,6 +271,18 @@ private void unzip(File zip, File dest) throws IOException {
         }
     }
 
+    /**
+     * Checks the entry name for zip-slip vulnerable sequences.
+     *
+     * @throws IllegalArgumentException if the entry contains vulnerable sequences
+     */
+    private String safeZipEntryName(String name) {
+        if (name.isEmpty() || name.contains("..") || name.startsWith("/") || name.startsWith("\\")) {
```

Comment:
Should we use `GUtil.isUnsafeZipEntryName()` here?

Reply:
The wrapper code cannot use `GUtil`.